README.md

```diff
@@ -358,6 +368,29 @@ Alternatively, authentication token can be set on a per-request basis, which als
   const img = await layer.getMap(getMapParams, ApiType.PROCESSING, requestsConfig);
 ```
 
+## Utility functions
+
+### Async conversion between Blob and Canvas
+
+Function `drawBlobOnCanvas` allows drawing a `Blob` on existing canvas element:
+
+```javascript
+  const blob = await layer.getMap(params, ApiType.WMS);
+
+  const canvas = document.createElement('canvas');
+  canvas.width = params.width;
+  canvas.height = params.height;
+  const ctx = canvas.getContext('2d');
+
+  await drawBlobOnCanvas(ctx, blob, 0, 0);
+```
+
+Function `canvasToBlob` converts an existing canvas to `Blob`:
```

Comment:
I would change this text to:

Function `canvasToBlob`  converts the provided canvas to `Blob`:


(since it accepts the `canvas` as a parameter)

Reply:
Thank you, your version is better, fixed.

Reply:
Resolving.